cmd/sync-tags/main.go

```diff
@@ -323,6 +306,31 @@ func main() {
 			glog.Fatalf("Failed to create tag %q: %v", bName, err)
 		}
 		createdTags = append(createdTags, bName)
+
+		// if publishSemverTags is true, create annotated semver tag
+		if *publishSemverTags {
```

Comment:
does this script sync *all* existing tags? will this try to create semver tags corresponding to all the existing kubernetes release tags?

Reply:
No, only newer tags but will make this simpler while fixing https://github.com/kubernetes/publishing-bot/pull/210#discussion_r354082521

Reply:
Updated.